CIP-1694/README.md

```diff
@@ -0,0 +1,1124 @@
+
+The constitutional committee can be replaced in one of two ways:
+
+* When in a **normal** state (i.e. a state of **confidence**), the committee can be replaced via a
+specific governance action (action 2 below) that requires the approval of both the **current constitutional committee** and the **DReps**.
+
+* When in a state of **no-confidence**, the committee can also be replaced via the same governance action (action 2 below),
+but this instead requires the approval of both the **SPOs** and the **DReps**.
+
+#### Size of the constitutional committee
+
+Unlike the Shelley governance design, the size of the constitutional committee is not fixed.
+It may be changed whenever a new committee is elected (action 2 below).
+Likewise, the committee _quorum_ (the number of committee `Yes` votes that are required to ratify governance actions) is not fixed and
+can also be varied by governance action 2.
+This gives a great deal of flexibility to the composition of the committee.
+
+#### Term limits
+
+
+The system will automatically enter a state of no-confidence when the term limit for the constitutional
+committee expires.  This limit is a governance protocol parameter, which specifies the maximum number of epochs
+during which the committee can ratify governance actions.  When the committee term limit expires, all governance
+actions will be dropped, and a new committee must be elected.
+
+The term limit will reset whenever governance action 2 is enacted, even if the same committee is re-elected
+and the quorum remains unchanged.  This allows Ada holders to confirm their confidence in the committee if they wish.
+Note that the term limit is calculated at the point the committee is elected.  Any change in the underlying protocol
+parameter only affects the term that applies to future committees.
+
+
+<!--------------------------- Constitutional committee ------------------------>
+<!---------------------------           DReps          ------------------------>
+
+### Delegated representatives (DReps)
+
+> **Warning**
+> CIP-1694 DReps **should not be conflated** with Project Catalyst DReps.
+
+<!-- TODO find another name that still points to liquid democracy. -->
+
+#### Pre-defined DReps
+
+In order to participate in governance, each stake credential must be delegated to a DRep.
+Ada holders will generally delegate their voting rights to a registered DRep
+that will vote on their behalf.  In addition, two pre-defined DRep options are available:
+
+* `Abstain`
+
+  If an Ada holder delegates to `Abstain`, then their stake is actively marked
+  as not participating in governance.
+
+  The effect of delegating to `Abstain` on chain is that the delegated stake *will not* be considered to be
+  a part of the active voting stake.  However, the stake *will* be considered to be registered for the
+  purpose of the incentives that are described in [Incentives to delegate voting stake](#incentives-to-delegate-voting-stake).
+
+* `No Confidence`
```

Comment:
I do not think this should be a default/pre-defined option. I see it as likely to create false-positives that have the risk of being used to the benefit of adversaries.

I think stakeholders can remain Abstained and if they strongly feel the need to vote a No Confidence then they can get active/delegate to a dRep that is aligned with their wishes.

OTOH: I see the simplicity in simply switching to a No Confidence pre-defined, but I also see the likelihood for users to leave it on there and forget, so unless a switchover system exists that moves it to Abstain after some period and back to No Confidence if say that happens to be the majority position of SPOs, then that would be more appropriate.

In conclusion, an Abstain pre-defined should be enough.

Reply:
I agree with your worries here. Note however that anyone can just make a script DRep that always votes like `No Confidence` here (so that everyone can vote for that DRep, but only in the way described). So the only real question here is: should a `No Confidence` DRep be more convenient (by being built-in & automatic) or not?

The same argument applies to the `Abstain` option of course, so we could remove both of them and replace them with scripts. Or you just delegate your vote to the all zero hash, that's equivalent to it as well (since it's safe to assume no-one has a key for that).

Reply:
I think keeping `Abstain` is safe as it is a neutral and has virtually no effect in the active votes. 

I also think that `No Confidence` should be dropped from being built-in & automatic. If a script is deployed by a third-party to represent `No Confidence` then at least a user will have to take a more elaborated path and set of steps to it.